manticore/platforms/linux.py

```diff
@@ -1716,8 +1726,16 @@ def sys_futex(self, uaddr, op, val, timeout, uaddr2, val3):
         logger.debug("sys_futex(...) -> -1")
         return -1
     def sys_getrlimit(self, resource, rlim):
-        logger.debug("sys_getrlimit(%x, %x) -> -1", resource, rlim)
-        return -1
+        ret = -1
+        if resource in self.rlimits:
+            rlimit_tup = self.rlimits[resource]
+            # 32 bit values on both 32 and 64 bit platforms. For more info,
+            # see the BUGS section in rlimit(2) man page.
```

Comment:
`man 2 rlimit` returns nothing for me on ubuntu 16.04, is this a typo?

Reply:
Sorry, meant `getrlimit(2)`